src/inmanta/app.py

```diff
@@ -1054,7 +1063,12 @@ def app() -> None:
 
     # Log config
     component = options.component if hasattr(options, "component") else None
-    log_config.apply_options(options, component, log_context)
+    if hasattr(options, "func") and options.func.__name__ == "default_logging_config":
+        # The logging config related CLI options, passed to the output-default-logging-config command, are intended for
+        # the generated logging config and not for the logging of the execution of the command itself.
+        log_config.apply_options(Options(), component=None, context=log_context)
```

Comment:
I really don't like this. I would change name of the internal/hidden option called 'component' to not have to do this. 

i.e. before, the component option was not set on the `default_logging_config` because it itself is not that component, I would like to keep this this way.